### ISSUE TITLE
how to refetch the tile image?

### ISSUE THREAD

**Issue by chengdonglin** [openseadragon/openseadragon#2602]
when view the level = 5, has any api to  refetch the tile image when click refetch button,

**Comment by iangilman** [comment]
Why do you want to refetch the tile? Has its content changed?

You can call `reset` on your tiled image to reload all of its tiles.

**Comment by chengdonglin** [comment]
> Why do you want to refetch the tile? Has its content changed?
> 
> You can call `reset` on your tiled image to reload all of its tiles.

yes, we need display different channel image data, reset command I think it can cause level and the coordinates of  viewer will changed.

**Comment by iangilman** [comment]
`reset` shouldn't change the viewport... you'll still be zoomed and panned to the same location.

**Comment by chengdonglin** [comment]
> `reset` shouldn't change the viewport... you'll still be zoomed and panned to the same location.

my openseadragon version is 2.x, I have found the reset below:
```js
    // deprecated
    reset: function() {
        $.console.error( "[Drawer.reset] this function is deprecated. Use World.resetItems instead." );
        this.viewer.world.resetItems();
        return this;
    },
```

when I call the reset api 
```js
Uncaught TypeError: this.viewer.reset is not a function
```
also I have try executor ```this.viewer.world.resetItems();``` it not refetch tile image

**Comment by iangilman** [comment]
Oh! Wow... 2.x was a long time ago. I don't remember what the API was like back then. Have you considered upgrading?

**Comment by chengdonglin** [comment]
> Oh! Wow... 2.x was a long time ago. I don't remember what the API was like back then. Have you considered upgrading?
hank you for your reply. I will review the source code.
I have try to update the version to 4.x, But we found the high version view not smooth, so I back to version2.x

**Comment by iangilman** [comment]
Interesting. Not smooth how? Like slower framerate? 

Anyway, it should be possible with 2.x but yeah, you'll have to look at the code to figure out what the API was. 

I've never really thought about what's involved in getting documentation for old versions. I suppose one option would be to grab the website from back then from this commit in the repository: 

https://github.com/openseadragon/openseadragon.github.com/tree/b5e9ea04d27bf07473fceb9e621756a73713b5cd

... and then you could run it locally and look at the documentation.
